src/coreclr/hosts/unixcoreruncommon/coreruncommon.cpp

```diff
@@ -215,6 +216,12 @@ int ExecuteManagedAssembly(
     coreClrDllPath.append("/");
     coreClrDllPath.append(coreClrDll);
 
+#if defined(__LINUX__)
+    std::string coreClrTPproviderDllPath(clrFilesAbsolutePath);
```

Comment:
Is the provider expected to be next to CoreCLR? If so, can you please also make the change to incorporate the same semantic in CoreCLR's nuget package?

@weshaggard should be able to give you pointers on it.